doc/api/process.md

```diff
@@ -1775,6 +1784,381 @@ Will generate an object similar to:
+-->
+
+* {number}
+
+The [`timeOrigin`][] specifies the high-resolution millisecond timestamp from
+which all performance metric durations are measured.
+
+### Class: PerformanceEntry
+<!-- YAML
+added: REPLACEME
+-->
+
+The base class for all objects in the Performance Timeline.
+
+#### performanceEntry.duration
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds ellapsed for this entry. This value will not
+be meaningful for all Performance Entry types.
+
+#### performanceEntry.name
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The name of the performance entry.
+
+#### performanceEntry.startTime
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp marking the starting time of the
+Performance Entry.
+
+#### performanceEntry.entryType
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The type of the performance entry. Current may be one of: `'node'`, `'mark'`,
+or `'measure'`.
+
+### Class: PerformanceFrame
+<!-- YAML
+added: REPLACEME
+-->
+
+A subclass of `PerformanceEntry` that provides timing metrics specific to the
+Node.js event loop.
+
+#### performanceFrame.duration
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds that have passed in the current loop.
+
+#### performanceFrame.prior
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds passed for the prior iteration of the loop.
+
+### Class: PerformanceNodeTiming
+<!-- YAML
+added: REPLACEME
+-->
+
+A subclass of `PerformanceEntry` that provides timing metrics specific to the
+Node.js runtime.
+
+#### performanceNodeTiming.arguments
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js command-line
+arguments processing completed.
+
+#### performanceNodeTiming.bootstrapComplete
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js process
+completed bootstrap.
+
+#### performanceNodeTiming.duration
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds that the Node.js process has been running.
+
+#### performanceNodeTiming.initialize
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js platform was
+initialized.
+
+#### performanceNodeTiming.inspectorStart
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js inspector
+agent completed startup.
+
+#### performanceNodeTiming.loopExit
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js event loop
+exited.
+
+#### performanceNodeTiming.loopStart
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js event loop
+started.
+
+### Class: process.PerformanceObserver(callback)
+
+* `callback` {Function}
```

Comment:
I am not sure, but maybe it is worth documenting the callback's arguments? Is `observer` the same as the `new process.PerformanceObserver()` here? And maybe [`PerformanceObserverEntryList`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserverEntryList) API is also worth documenting?